CODING_STANDARD.md

```diff
@@ -19,7 +19,9 @@ Formatting is enforced using clang-format. For more information about this, see
       - Nested function calls do not need to be broken up into separate lines
         even if the outer function call does.
 - If a method is bigger than 50 lines, break it into parts.
-- Put matching `{ }` into the same column.
+- Put matching `{ }` into the same column, except for lambdas, where you should
+  place `{` directly after the closing `)`. This rule also doesn't apply to
+  initializer lists.
```

Comment:
Could you add a comment on the rationale of this (convenience rule due to deficiencies of clang-format)? We should revert the rule once the clang-format bug is fixed.

Reply:
`This rule also doesn't apply to initializer lists.` sounds a bit strange to me. I'd remove the `also` - is it the rule (Put matching...) or the exception (except...) that doesn't apply... a bit confusing. Please reword.